package.json

```diff
@@ -33,18 +35,24 @@
     ]
   },
   "scripts": {
-    "build": "tsc && node-pre-gyp rebuild",
+    "build": "tsc && node-pre-gyp rebuild && tsc -p tsconfig.browser.json",
     "test": "node --expose-gc dist/test/test-load.js",
-    "install": "tsc && node-pre-gyp install --fallback-to-build"
+    "install": "tsc && node-pre-gyp install --fallback-to-build && tsc -p tsconfig.browser.json"
   },
   "devDependencies": {
+    "@types/crypto-js": "^3.1.43",
     "@types/node": "^10.12.18",
+    "@types/ws": "^6.0.1",
     "aws-sdk": "^2.419.0",
     "node-gyp": "^3.8.0",
-    "typescript": "^3.2.2",
+    "typescript": "^3.5.3",
     "yargs": "^13.2.4"
   },
   "dependencies": {
-    "node-pre-gyp": "^0.12.0"
+    "async-mqtt": "^2.3.0",
+    "crypto-js": "^3.1.9-1",
+    "mqtt": "^2.3.1",
```

Comment:
reeeeeeeally wish there was a way to do browser-only dependencies

Reply:
ME TOO. I TRIED!